styles/stylesheet.css

```diff
@@ -0,0 +1,135 @@
+z* {
+  border: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html[Attributes Style] {
+    -webkit-locale: "en";
+    direction: ltr;
+    unicode-bidi: isolate;
+}
+
+html {
+  display: block;
+}
+
+body {
+    font-family: "Arial", sans-serif;
+    max-width: 100%;
+    display: block;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  margin: 0;
+  flex-shrink: 0;
+  flex-wrap: wrap;
+  padding: 3em 4em 0;
+  background-image: url(https://imgur.com/364pi2q.png);
+  background-size: cover;
+  color: #FFFFFF;
+  height: 36em;
+}
+
+.headertext {
+  font-size: 1.9rem;
+  font-style: italic;
+  color: green;
+}
+
+.checkoutbox {
+  margin: 0;
+  display: block;
+}
+
+.logoimg {
+  width: 35em;
+}
+
+.cartimg {
+  width: 2em;
+  margin-right: .5rem;
+}
+
+.checkoutbutton {
+    background-color: green;
+    color: #FFFFFF;
+    height: 3em;
+    width: 100%;
+    padding: 1em 2em;
+    border-radius: 0.2em;
+}
+
+/* NAVIGATION */
+
+nav {
+  background-color: #d1d1d1;
+  height: 3.9rem;
+}
+
+nav ul {
+  padding: 18px 0;
+  width: 90%;
+  margin: auto;
+}
+
+li {
+  padding: 18px 0;
+  display: inline;
+  list-style-type: none;
+  text-decoration: none;
+}
+
+li a {
+  text-decoration: none;
+  color: #1c4282;
+  font-size: 1.1rem;
+  font-weight: bold;
+  padding: 1em;
+}
+
+/* MAIN SECTION */
+
+.mainsection {
+    width: 90%;
+    margin: 20px auto;
+    display: flex;
+}
+
+article {
+  display: block;
+  width: 50%;
+  padding: 2em;
+}
+
+h2 {
+  padding-bottom: 1em;
+  color: #1c4282;
+}
+
+/* DOG PROFILESf */
+
+.dogprofiles {
+  display: flex;
+  width: 90%;
+  margin: 20px auto;
+  justify-content: center;
+  align-content: space-between;
+  flex-direction: row;
+  flex-flow: wrap;
+  align-items: center;
+}
+
+.dog {
+  display: block;
+  width: 22em;
```

Comment:
These aren't exactly responsive. I'd lean you to using some sort of percent based system to keep them behaving properly. In your page I only see 2 dog tiles wide on a laptop, and we should see three. It also keeps them in ration with the rest of the page if you go with a percent based system.

Reply:
Okay! I shall work on integrating that when I work on the next assignment!